packages/main/src/themes/TableRowBase.css

```diff
@@ -20,6 +18,17 @@
 	clip-path: inset(var(--sapContent_FocusWidth)); /* focus outline should not overlap sticky cells */
 }
 
+::slotted(ui5-table-cell:first-of-type),
+::slotted(ui5-table-header-cell:first-of-type),
+#popin-cell {
+	padding-left: var(--_ui5_first_table_cell_horizontal_padding);
+}
+
+[ui5-table-cell-base] + ::slotted([ui5-table-cell-base]:first-of-type),
+[ui5-table-cell-base] ~ #popin-cell {

```

Comment:
🤔 isn't #popin-cell always before the other cells?

Reply:
Changed it to #selection-cell to be explicit